Modules/Common/src/TH1SliceReductor.cxx

```diff
@@ -141,7 +141,7 @@ void TH1SliceReductor::GetTH1StatsY(TH1* hist, float stats[3],
     ILOG(Error, Support) << "Error: Negative bin in TH1ReducterTPC::GetTH1StatsY" << ENDM;
     exit(0);
   }
-  if (upperBin <= lowerBin) {
```

Comment:
Please remove this, I will update the TH1SliceReductor in the next couple of days as there are more changes to be made

Reply:
Hi @mlesch , 

Thanks a lot for the review!

Yes, I tested the changes using the in a postprocessing of apass2 data from LHC22m on EMCAL output.

OK, I will revert the changes in the TH1SliceReductor.